src/main/java/org/jenkinsci/plugins/workflow/graphanalysis/ForkScanner.java

```diff
@@ -406,6 +470,13 @@ protected FlowNode hitParallelStart() {
     }
 
     @Override
+    public FlowNode next() {
```

Comment:
Yes, good point (probably for all of them, and worth a mention in AbstractFlowScanner Javadocs).   Waiting for the review batch to be in to batch up changes though.